js/src/wrappers/ai-sdk.ts

```diff
@@ -24,10 +25,17 @@ import {
  * Wrap an ai-sdk model (created with `.chat()`, `.completion()`, etc.) to add tracing. If Braintrust is
  * not configured, this is a no-op
  *
- * @param model
+ * @param model The AI SDK model to wrap.
+ * @param overrideSpan Optional. A Braintrust Span object to use for tracing this model's operations.
+ *                     If provided, this span will be used instead of creating a new one internally.
+ *                     The lifecycle of this span (e.g., calling .end()) should be managed by the caller
+ *                     if it's passed here.
  * @returns The wrapped object.
  */
-export function wrapAISDKModel<T extends object>(model: T): T {
+export function wrapAISDKModel<T extends object>(
+  model: T,
+  overrideSpan?: Span,
```

Comment:
would be nice to have an opts argument that is an object so we can add more options over time. So something like

```
export interface WrapAISDKModelOpts {
  span?: Span
}

export function wrapAISDKModel<T extends object>(model, opts?: WrapAISDKModelOpts) {
  opts = opts ?? {};
  ...
}